src/enterprise/extension.ts

```diff
@@ -146,6 +148,13 @@ function registerAuthenticationProviders(
     ),
     provider
   );
+
+  context.subscriptions.push(
+    commands.registerCommand(
+      SIGN_IN_AUTH_TOKEN_COMMAND,
+      SignInUsingCustomTokenCommand
+    )
+  );
```

Comment:
not familiar with the arch, so I'll just ask, :) 
why do we need to register twice? (commandHandler and extension?)